CONTRIBUTING.md

```diff
@@ -28,14 +30,10 @@ npx playwright install
 
 Playwright is a multi-package repository that uses npm workspaces. For browser APIs, look at [`packages/playwright-core`](https://github.com/microsoft/playwright/blob/main/packages/playwright-core). For test runner, see [`packages/playwright`](https://github.com/microsoft/playwright/blob/main/packages/playwright).
 
-Note that some files are generated by the build, so the watch process might override your changes if done in the wrong file. For example, TypeScript types for the API are generated from the [`docs/src`](https://github.com/microsoft/playwright/blob/main/docs/src).
+### TypeScript Type Generation
```

Comment:
I guess it makes sense to put this after `Write documentation`. Or perhaps combine them? As you put it here, it seems like the most important part of making a change is to "generate types". In practice, you edit docs and everything just happens automatically 😄